.github/workflows/build.yml

```diff
@@ -164,6 +164,24 @@ jobs:
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
+  android-test:
+    name: "Android: instrumented tests"  
+    strategy:
+      matrix:
+        api-level: [21, 24, 26, 29]      
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2    
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8    
+    - name: Run instrumented tests
+      uses: reactivecircus/android-emulator-runner@v2
+      with:        
+        api-level: ${{ matrix.api-level }}
+        script: cd android/ && sh gradlew --no-daemon clean build connectedCheck --info
+        arch: x86_64
```

Comment:
Can we run this as part of the `android` job? What's the difference between all the API levels and do we really need to test them all?

Reply:
We should test 21 and 29 at least. It can't be a part of android as it runs against both mac and windows (hardware acceleration is available only on mac).

Reply:
We can leave only 29 if CI times are more important rather than possibility of breaking something on v21 (which is not very likely taking into account the subset of android APIs that we're using).